core/src/autogluon/core/trainer/abstract_trainer.py

```diff
@@ -871,6 +872,15 @@ def get_model_pred_proba_dict(self,
         else:
             return model_pred_proba_dict
 
+    def get_model_oof_dict(self, models: List[str]) -> dict:
+        """
+        Returns a dictionary of out-of-fold prediction probabilities, keyed by model name
+        """
+        model_oof_dict = {}
+        for model in models:
+            model_oof_dict[model] = self.get_model_oof(model=model)
+        return model_oof_dict
```

Comment:
A dictionary comprehension would be a pythonic way to accomplish this if you are interested:
`return {model: self.get_model_oof(model) for model in models}`

Reply:
good callout, updated